components/server/src/workspace/workspace-starter.ts

```diff
@@ -838,6 +845,20 @@ export class WorkspaceStarter {
                 usageAttributionId,
                 workspaceClass,
             };
+
+            if (usageAttributionId) {
```

Comment:
@laushinka, I wonder if that call is placed right in `newInstance`, which just pumps up the instance object, but does not do any side-effects otherwise. 

maybe that should be part of `actuallyStartWorkspace`, preferably after the workspace creation request (to wsman) was already done, so it's not gonna be delayed further.

Reply:
Hmm, I guess it makes sense because this call isn't supposed to block or exit the ongoing process.

Reply:
@AlexTugarev Change done - moved to workspace creation request. Tested and still works fine 👍🏼